migrations/2022-09-08-102358_site-and-community-languages/down.sql

```diff
@@ -0,0 +1,2 @@
+drop table site_language;
+drop table community_language;
```

Comment:
diesel migration redo works fine. Problem here is that local_user_language table etc were added in an earlier migration. Maybe it would make sense to move the update function there?

Reply:
You need to undo those inserts you did in the up.sql (IE delete). Ideally every migration should be reversible.

Reply:
So just delete everything in those tables? Wouldnt it be better in this case to move these update functions into the older migrations where tables local_user_language etc were created?

Reply:
also done